### PR TITLE
initial impl guide and starting to pull informative info into their a…

### PR DESCRIPTION
**Pull request by 2byrds**
…nd reference it

### CODE REVIEW

**Comment by 2byrds** [comment]
@daidoji I completed a full pass on the spec and pulled items into the implementors guide.

**Comment by 2byrds** [comment]
@daidoji could you please review/approve this PR?

File: spec/core.md [hunk 1]
```diff
@@ -65,7 +63,8 @@ This section is normative.
     1. The [[ref: KERI event stream]] MUST be [[ref: CESR]]-formatted (media type of application/cesr) and the KERI events must be verifiable using the KERI rules.
 1. The `did:web` version of the DIDs MUST be the same (minus the `s`) and point to the same `did.json` file, but have no knowledge of the `keri.cesr` file.
 
-> The set of KERI features needed for most `did:webs` use cases is modest, with limited dependencies. These basics are summarized in the [KERI Fundamentals](#keri-fundamentals) section of this specification. This specification assumes a working knowledge of the concepts there. The inclusion of KERI in `did:webs` enables a number of capabilities for securing a `did:webs` identifier, including multi-signature support and the creation of [[ref: pre-rotated]] keys to prevent loss of control of the identifier if the current private key were to be compromised.
+For more information, see the following sections in the implementators guide:
```
Comment (daidoji) [inline spec/core.md:66]:
```suggestion
For more information, see the following sections in the implementors guide:
```

**Review by daidoji — APPROVED** [review]
Excepting the typo this looks fine.

**Comment by 2byrds** [comment]
> Excepting the typo this looks fine.  

U don't like taters? Lol, good catch thx @daidoji

**Pull request merged**